### PR TITLE
Fix tarball build and refresh monolite due to memory model change

### PR DESCRIPTION
**Pull request by jaykrell**
Tarball build was broken by https://github.com/mono/mono/commit/9010d2f290d28dc97ac276e0c0d32a2932b7550f

### CODE REVIEW

**Comment by jaykrell** [comment]
@monojenkins build deb with monolite

**Comment by jaykrell** [comment]
@monojenkins build failed

**Comment by jaykrell** [comment]
@monojenkins build deb with monolite

**Comment by jaykrell** [comment]
@monojenkins build deb with monolite

**Comment by jaykrell** [comment]
Might have to undo other, or combine that with this. I not at computer to read errors.

**Comment by jaykrell** [comment]
@monojenkins build deb with monolite

**Comment by jaykrell** [comment]
@monojenkins build deb with monolite

**Comment by jaykrell** [comment]
@monojenkins build deb with monolite

**Comment by jaykrell** [comment]
@monojenkins build deb with monolite

**Comment by jaykrell** [comment]
@monojenkins build deb with monolite

**Comment by akoeplinger** [comment]
@monojenkins build deb with monolite

**Comment by jaykrell** [comment]
@monojenkins build deb with monolite

**Comment by akoeplinger** [comment]
@monojenkins build deb with monolite

**Comment by jaykrell** [comment]
@monojenkins build deb with monolite

**Comment by akoeplinger** [comment]
@monojenkins build deb with monolite

**Comment by akoeplinger** [comment]
@monojenkins build deb with monolite

**Comment by akoeplinger** [comment]
@monojenkins build failed

**Comment by akoeplinger** [comment]
@monojenkins build deb with monolite

**Comment by jaykrell** [comment]
@monojenkins build failed

File: mcs/build/common/basic-profile-check.cs [hunk 1]
```diff
@@ -41,9 +41,9 @@ static int Main ()
 
 		Version min_mono_version;
 #if __MonoCS__
-		min_mono_version = new Version (6, 2);
+		min_mono_version = new Version (6, 5);
```
Comment (jaykrell) [inline mcs/build/common/basic-profile-check.cs]: Remove if?

Comment (akoeplinger) [reply mcs/build/common/basic-profile-check.cs]: Hm this is for cases where CSC needs a newer/different mono than MCS, so I think it's fine to keep it.

**Pull request merged**